sdk/servicelinker/azure-resourcemanager-servicelinker/src/samples/README.md

```diff
@@ -0,0 +1,57 @@
+<!-- ---
+page_type: sample
+languages:
+  - java
+products:
+  - azure
+  - azure-service-linker
+urlFragment: service-linker-resourcemanager-samples
+--- -->
+
+# Azure Service Linker Create Connection Samples for Java
+
+This contains Java samples for create service connection using Service Linker Provider.
+
+## Getting started
+
+Getting started explained in detail [here][SERVICELINKER_README_GETTING_STARTED].
+Please refer it to add dependency and configure authentication environment variables.
+
+## Samples Details
+
+All samples are located in [CreateServiceLinker][SERVICELINKER_SAMPLE_CODE] file, you can use `main()` entrypoint to run it directly.
+
+### CreatSpringCloudAndSQLConnection
```

Comment:
since this is java sample, I think we can add more springcloud samples, such as from data, mysql+spring is used a lot. this can be done in later pr

Reply:
we could discuss later.